uint32 id
geometry_msgs/Point start_point
geometry_msgs/Point end_point
---
bool status

